test/cypress/integration/widgets/deployment_button_spec.ts

```diff
@@ -1,9 +1,10 @@
-// @ts-nocheck File not migrated fully to TS
 describe('Create Deployment Button widget', () => {
     const resourcePrefix = 'deploy_test_';
     const testBlueprintId = `${resourcePrefix}bp`;
     const requiredSecretsBlueprint = `${resourcePrefix}required_secrets_type`;
     const customInstallWorkflowBlueprint = `${resourcePrefix}custom_install_workflow_type`;
+    const lablesBlueprintWithoutPrefix = 'labels';
```

Comment:
Minor thing, but let's fix at once:
```suggestion
    const labelsBlueprintWithoutPrefix = 'labels';
```